web-animations/interfaces/KeyframeEffect/keyframe-properties.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<meta charset='utf-8'>
+<title>Web Animations API: Keyframe Property tests</title>
+<link rel='help' href='https://w3c.github.io/web-animations/#processing-a-keyframes-argument'>
+<script src='/resources/testharness.js'></script>
+<script src='/resources/testharnessreport.js'></script>
+<script src='../../testcommon.js'></script>
+<body>
+<script>
+'use strict';
+
+test(function(t) {
+  var keyframe = {};
+  Object.defineProperty(keyframe, 'width', {value: '200px'});
+  Object.defineProperty(keyframe, 'height', {
+    value: '100px',
+    enumerable: true});
+  assert_equals(keyframe.width, '200px', 'width of keyframe is readable');
+  assert_equals(keyframe.height, '100px', 'height of keyframe is readable');
+  try {
+    createDiv(t).animate([keyframe, {height: '200px'}], 1);
+  } catch (e) {
+    assert_unreached('Mismatched properties - both or neither properties on keyframe were considered.');
```

Comment:
Is this relying on Blink stable throwing when a synthetic keyframe is used? I can't see why a try catch is used here otherwise.
This should use assert_frame_lists_equal(effect.getKeyframes(), ...) from keyframe-utils.js.